sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Bindings/WebPubSubContextJsonConverter.cs

```diff
@@ -13,13 +13,18 @@ namespace Microsoft.Azure.WebJobs.Extensions.WebPubSub
 {
     internal class WebPubSubContextJsonConverter : JsonConverter<WebPubSubContext>
     {
+        public override bool CanRead => false;
+        public override bool CanWrite => true;
+
         public override WebPubSubContext ReadJson(JsonReader reader, Type objectType, WebPubSubContext existingValue, bool hasExistingValue, JsonSerializer serializer)
         {
             throw new NotImplementedException();
         }
 
         public override void WriteJson(JsonWriter writer, WebPubSubContext value, JsonSerializer serializer)
         {
+            // overwrite customers json converter.
+            serializer = WebPubSubJsonSerializer.Serializer;
```

Comment:
Is there a reason we need to ignore customer serializer as opposed to just adding the converters that we need?

Reply:
Not need. The serialize place should ensure to add required serializer.